source/features/forked-to.tsx

```diff
@@ -47,22 +47,30 @@ async function updateUI(forks: string[]): Promise<void> {
 	}
 
 	document.body.classList.add('rgh-forked-to');
-	const forkCounter = await elementReady('.social-count[href$="/network/members"]', {waitForChildren: false});
+	const forkCounter = select('#repo-network-counter');
+	const forkBoxContents = forkCounter!.parentElement!;
+	const forkBox = forkBoxContents.parentElement!;
+	const forkContainer = forkBox.parentElement!;
```

Comment:
Messy... will try to find better selectors

Reply:
`forkContainer` is now `.pagehead-actions > li > div.float-left` (the other `li`s have `a`, `notification-list-subscription-form`, and `div` with various `js-*` classes as the only child element). Is that better?

Couldn't find anything too deterministic for `forkBox` (`div.float-left > details:nth-child(1)` inside `forkContainer`)  and `forkBoxContents` (`summary` inside `forkBox`).